### PR TITLE
Updating HA in Canton

### PR DESCRIPTION
**Pull request by carrielaben-da**
Addressing Soren's comments from previous PR [167](https://github.com/digital-asset/docs.daml.com/pull/167
)
Diagrams still need to be updated.

### CODE REVIEW

**Comment by garyverhaegen-da** [comment]
LGTM but I don't know the subject well enough, so I'll let @soren-da stamp.

Do we want to propagate those changes to 2.6? (once they're finalized; no need to double the work while we're still tweaking things)

**Review by loiskilberg-da — APPROVED** [review]

**Comment by carrielaben-da** [comment]
> LGTM but I don't know the subject well enough, so I'll let @soren-da stamp.
> 
> Do we want to propagate those changes to 2.6? (once they're finalized; no need to double the work while we're still tweaking things)

I believe so, yeah.

File: docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst [hunk 1]
```diff
@@ -38,18 +38,18 @@ The sequencer relies on the database for both HA and consistency. The database e
 Many sequencer nodes can be deployed. Each node has concurrent read and write components when accessing the database. The load balancer evenly distributes requests between sequencer nodes.
 
 .. NOTE::
-   The system halts requests to a node whose status is reported as unhealthy by the health endpoint.
+   The system stops sending requests to an unhealthy node.
 
-Ensuring Consistency
-````````````````````
+Consistency and the Database Sequencer
+``````````````````````````````````````
```
Comment (soren-da) [inline docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst:44]: how about making this one and the next a subsection of Database Sequencer?

File: docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/overview.rst [hunk 2]
```diff
@@ -8,23 +8,17 @@ Overview
...
+    * Sequencer
 
-    * Mediator.
+    * Mediator
 
-    * Topology manager.
+    * Domain manager
 
-    * Domain service.
+    * Domain service
```
Comment (soren-da) [inline docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/overview.rst:21]: this component does not exist anymore, you can remove it

**Review by soren-da — APPROVED** [review]
Looks good!

File: docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst (hunk 1 above)
Comment (carrielaben-da) [reply docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/domain.rst:44]: I'll put this in the next PR with the updated diagrams.

File: docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/overview.rst (hunk 2 above)
Comment (carrielaben-da) [reply docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/overview.rst:21]: Same as above, I'll incorporate this in the next PR.

**Pull request merged**